### ISSUE TITLE
unable to build rmf_fleet_adapter

### ISSUE THREAD

**Issue by cwrx777** [osrf/rmf_core#70]
when trying to build rmf_fleet_adapter, error encountered:  Cannot locate dosdep defintion for [rmf_fleet_utils]. 

In rmf_fleet_adapter package.xml,  there is dependency with rmf_fleet_utils.
  <depend>rmf_fleet_utils</depend>

Am I missing something?

my envinroment:
ubuntu 18.04.3
eloquence patch 1.
I run performed the following:
1. git clone rmf_core into a folder, e.g. rmf_core
2. from the parent folder of rmf_core, i run:
sudo rosdep install -i --from-path rmf_core --rosdistro eloquent -y  --ignore
result:
rmf_fleet_adapter: Cannot lcate rosdep definition for [rmf_fleet_utils]

is there any guide on how to build and use this?

**Comment by mxgrey** [comment]
`rmf_fleet_utils` shouldn't be in the `package.xml` anymore; that's out of date.

I think colcon should still be able to build everything; you'll just need to ignore the rosdep complaint until we fix the manifest file.

Unfortunately instructions do not currently exist because this repo is under heavy heavy :construction: construction :construction:. I would recommend holding off on writing any code against it for roughly 1-2 more months, at which point I think the API will be much more stable.

**Comment by cwrx777** [comment]
Hi @mxgrey Thanks for your reply. I am a newbie, trying to understand how rmf_core works.
I tried building with colcon and encountered error when building rmf_traffic that it was unable to find 'fcl' package. 
is there a list of depencies I need to install?
is rmf_core compatible with dashing or eloquent?

**Comment by mxgrey** [comment]
> it was unable to find 'fcl' package

You can get that with `sudo apt install libfcl-dev`. I think we may need to add it as a `<build_depend>` to `rmf_traffic`.

**Comment by cwrx777** [comment]
Hi @mxgrey ,

i managed to build after installing fcl and yaml-cpp, with some warnings in rmf_fleet_adapter

```
sudo apt install libfcl-dev
sudo apt install libyaml-cpp-dev
```

**Comment by mikaelarguedas** [comment]
This is apparently still an issue https://answers.ros.org/question/343547/how-to-compile-rmf_core-using-colcon-build-in-ros2-eloquent/
Could this be reopened for visibility until it is resolved ?

**Comment by mxgrey** [comment]
This issue should now be resolved by #80

Feel free to reopen if there are still issues with building.
